core/src/main/java/com/rudderstack/core/internal/KotlinLogger.kt

```diff
@@ -18,7 +18,7 @@ import com.rudderstack.core.Logger
 
 object KotlinLogger : Logger {
```

Comment:
Let's align `KotlinLogger` constructor with the `AndroidLogger.kt `constructor, where you now have the `logLevel` parameter. Check also comment above.